### PR TITLE
Added Grey's Anatomy, John Wick 3, Jp, Narcos, BBT

### PR DESCRIPTION
**Pull request by unikornintech**
- [x] 🔍 I have searched the `data.json` file and confirmed I am not adding a duplicate entry.
- [x] 🌈 If I am adding a different version of a movie or show (for example Beauty and the Beast has two movie versions (2017 and 1999) and was also a broadway musical) among the emojis I have chosen, at least one is different from the previous version(s) that have been added.
- [x] 3️⃣ I have at least three emojis listed under `emojiImgs` and 5️⃣ I have less than five emojis listed under `emojiImgs`.
- [x] 👍 My pull request has a descriptive title (such as `Added The Lion King` or `Added Black Panther, The Avengers: Endgame and Thor`)
- [x] ⭐ My genres are each individually wrapped in quotation marks and have a comma between each one. (such as submitting this `"genres": ["adventure","mystery","animation"]` and not this `"genres":["adventure, mystery, animation"]`)
- [x] 🖍️ I have placed the new show(s) or movie(s) in alphabetical order based on title. 

👋 If this pull request closes an issue, replace the --- with the issue number.

Closes #---

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -467,5 +467,57 @@
...
-  }
+  },
+  {
+   "title": "Grey's Anatomy",
+   "emojiImgs": "💊💉🧪⚕🍷",
+    "genres": [
+          "drama",
+          "romance"
+      ],
+      "type": "TV Series",
```
Comment (brittanyrw) [inline data.json]: Looks good, just noticed a couple of things that need to be updated! The types should be `tv`, `movie` or `musical`. Please change the ones listed as `TV Series` to `tv` instead. This field is going to be used for filtering so all TV entries need to match.

File: data.json [hunk 2]
```diff
@@ -467,5 +467,57 @@
...
+      "year": 2005 -
```
Comment (brittanyrw) [inline data.json]: You have an extra dash here that needs to be deleted.

File: data.json [hunk 3]
```diff
@@ -467,5 +467,57 @@
...
+      },
+  {
+   "title": "Narcos",
+   "emojiImgs": "🔫🗡💊💉💰",
+    "genres": [
+          "crime",
+          "drama"
+      ],
+      "type": "TV Series",
+      "year": 2015-2017
```
Comment (brittanyrw) [inline data.json]: The year should only have one year (the start year). Please delete the dash and the ending year for the series you have added.

**Comment by brittanyrw** [comment]
@unikornintech Great job and thanks for adding so many new items! I noticed a couple of updates that need to be made and I added comments about them in the code. Once you make those updates we can go ahead and accept your additions.  🎉

**Review by unikornintech — COMMENTED** [review]
update to type and year

**Comment by brittanyrw** [comment]
Hi @unikornintech! Sorry for the delay, I missed your updates. Thanks for contributing to EmojiScreen!  🥳I have merged your changes in and you should be able to see them on the live site (emojiscreen.com) today. Feel free to contribute again in the future! 

If you'd like to contribute again: I changed the structure of the app a little bit so you'd need to pull down the changes from this project into your fork. Here are instructions for that: https://help.github.com/en/articles/syncing-a-fork Or you can just delete your current fork and make another one. 🙂

**Pull request merged**